pyterrier/utils.py

```diff
@@ -220,6 +220,26 @@ def convert_qrels_to_dict(df):
             run_dict_pytrec_eval[row.qid][row.docno] = int(row.label)
         return(run_dict_pytrec_eval)
 
+    @staticmethod
+    def convert_qrels_to_dataframe(qrels_dict) -> pd.DataFrame:
+        """
+        Convert a qrels dataframe to dictionary for use in pytrec_eval
```

Comment:
Convert a qrels dataframe to dictionary
->
Convert a qrels dictionary to dataframe